src/kerberos.cc

```diff
@@ -0,0 +1,221 @@
+#include <memory>
+
+#include "common.h"
+#include "kerberos.h"
+#include "kerberos_client.h"
+#include "kerberos_server.h"
+
+using v8::FunctionTemplate;
+
+#define GSS_MECH_OID_KRB5 9
+#define GSS_MECH_OID_SPNEGO 6
+
+static char krb5_mech_oid_bytes[] = "\x2a\x86\x48\x86\xf7\x12\x01\x02\x02";
+gss_OID_desc krb5_mech_oid = {9, &krb5_mech_oid_bytes};
+
+static char spnego_mech_oid_bytes[] = "\x2b\x06\x01\x05\x05\x02";
+gss_OID_desc spnego_mech_oid = {6, &spnego_mech_oid_bytes};
+
```

Comment:
Is it worth anonymous namespacing all the classes? Or is no one gonna pull this in like that?